components/organisms/SignUpForm/SignUpForm.js

```diff
@@ -1,16 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
 
 import SignUpFormContent from './SignUpFormContent';
 import useSignUp from './useSignUp';
 
+export const StyledFormWrapper = styled.div`
+  text-align: center;
+`;
+
+export const StyledToggleForm = styled.div`
+  display: inline-block;
+  cursor: pointer;
+  color: #00f;
+  margin-top: 1rem;
+`;
+
 const SignUpForm = ({ onSuccess }) => {
   // todo remove callbacks from mutation hooks
   const {
     signUp,
     mutationResult: { error, loading },
   } = useSignUp({ onSuccess });
 
-  return <SignUpFormContent error={error} loading={loading} onSubmit={signUp} />;
+  const [activeForm, toggleSignForm] = useState('signUp');
+  const handleActiveFormChoose = () => {
+    activeForm === 'signUp' ? toggleSignForm('signIn') : toggleSignForm('signUp');
```

Comment:
Let's define `signUp` `signIn` strings as constants outside of declaration component:
```
const SIGN_IN_FORM = 'signIn';
const SIGN_UP_FORM = 'signUp';
```

Then import them and use in the related `SignUpFormContent`, to clarify dependencies between this components

Reply:
Do you say about relocate them in vars.js file?